externs/pictureinpicture.js

```diff
@@ -42,3 +46,27 @@ HTMLMediaElement.prototype.requestPictureInPicture = function() {};
 
 /** @type {boolean} */
 HTMLMediaElement.prototype.disablePictureInPicture;
+
+
+/** @type {Function} */
+HTMLMediaElement.prototype.webkitSetPresentationMode;
+
+
+/** @type {Function} */
```

Comment:
Instead of Function, these should have a specific function type with arguments and a return value.  For example:

```js
/** @type {function(string):boolean} */
```

Or better yet, define the actual function and annotate with `@param` and `@return`, so that we can see a meaningful name for the parameters:

```js
/**
 * @param {string} mode
 * @return {boolean}
 */
HTMLMediaElement.prototype.webkitSupportsPresentationMode = function(mode) {};
```

The same goes for `webkitSetPresentationMode` above.